app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations, division
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, int | float):
```

Comment:
```suggestion
        if isinstance(other, (int, float)):
```
The correct way to use `isinstance`

Reply:
fix everywhere